### DISCUSSION TITLE
tutorial/formaction/base

### DISCUSSION THREAD

**Discussion by giscus** [yangbo5207/react19feature#35] [General]
# tutorial/formaction/base

http://localhost:5173/tutorial/formaction/base?fname=Jake%E6%92%92%E6%97%A6%E5%8F%91%E7%94%9F%E7%9A%84%E5%8F%91&lname=Ma

**Comment by yangbo5207** [comment]
欢迎各位道友添加我的微信好友 icanmeetu 交流学习心得
